packages/astro/src/vite-plugin-build-css/index.ts

```diff
@@ -73,6 +74,11 @@ export function rollupPluginAstroBuildCSS(options: PluginOptions): VitePlugin {
         delete viteCSSPost.renderChunk;
         delete viteCSSPost.generateBundle;
       }
+
+      const viteVue = resolvedConfig.plugins.find((p) => p.name === 'vite:vue');
+      if(viteVue) {
+        vueTransform = viteVue.transform;
+      }
```

Comment:
This begs the question, what if another plugin handles CSS? Will we need to special-case all of them here?

If there's a safer way to call any possible transforms? Like passing `viteServer` into this plugin and calling `viteServer.pluginContainer.transform()`? I'd be in favor of exploring that, but you have more context here.

Reply:
That's worth a try.